Drop the hard coded "gcc", also remove the -Werror since there are warnings with GCC 
author: fenglai liu (fenglai@accre.vanderbilt.edu)

diff -Naur snap_ori/Makefile snap/Makefile
--- snap_ori/Makefile	2019-03-20 12:14:03.152600843 -0500
+++ snap/Makefile	2019-03-20 13:54:45.172606540 -0500
@@ -77,7 +77,7 @@
 	cd Zoe; make clean
 
 depend: $(OBJECTS:.o=.c)
-	gcc $(INC) -MM $^ > $@
+	$(CC) $(INC) -MM $^ > $@
 
 tar:
 	rm -rf /tmp/$(APP)
@@ -94,11 +94,11 @@
 
 gcc:
 	cd Zoe; make;
-	make $(APP)  CC="gcc" CFLAGS="-O2 -Wall -Werror"
-	make $(APP2) CC="gcc" CFLAGS="-O2 -Wall -Werror"
-	make $(APP3) CC="gcc" CFLAGS="-O2 -Wall -Werror"
-	make $(APP4) CC="gcc" CFLAGS="-O2 -Wall -Werror"
-	make $(APP5) CC="gcc" CFLAGS="-O2 -Wall -Werror"
+	make $(APP)   CFLAGS="-O2 -Wall "
+	make $(APP2)  CFLAGS="-O2 -Wall "
+	make $(APP3)  CFLAGS="-O2 -Wall "
+	make $(APP4)  CFLAGS="-O2 -Wall "
+	make $(APP5)  CFLAGS="-O2 -Wall "
 
 
 ###################
diff -Naur snap_ori/Zoe/Makefile snap/Zoe/Makefile
--- snap_ori/Zoe/Makefile	2019-03-20 12:14:03.148600924 -0500
+++ snap/Zoe/Makefile	2019-03-20 13:55:51.632260071 -0500
@@ -76,7 +76,7 @@
 	rm -f *.o $(APP) $(APP2) $(APP3) $(APP4) $(APP5)
 
 depend: $(OBJECTS:.o=.c)
-	gcc -MM $^ > $@
+	$(CC) -MM $^ > $@
 
 tar:
 	rm -rf /tmp/$(APP)
@@ -92,14 +92,14 @@
 #################
 
 gcc:
-	make $(APP)  CC="gcc" CFLAGS="-O2 -Wall -Werror"
+	make $(APP)  CFLAGS="-O2 -Wall "
 
 all:
-	make $(APP)  CC="gcc" CFLAGS="-O2 -Wall -Werror"
-	make $(APP2) CC="gcc" CFLAGS="-O2 -Wall -Werror"
-	make $(APP3) CC="gcc" CFLAGS="-O2 -Wall -Werror"
-	make $(APP4) CC="gcc" CFLAGS="-O2 -Wall -Werror"
-	make $(APP5) CC="gcc" CFLAGS="-O2 -Wall -Werror"
+	make $(APP)  CFLAGS="-O2 -Wall "
+	make $(APP2) CFLAGS="-O2 -Wall "
+	make $(APP3) CFLAGS="-O2 -Wall "
+	make $(APP4) CFLAGS="-O2 -Wall "
+	make $(APP5) CFLAGS="-O2 -Wall "
 
 
 ###################
